LaTeX/DocStrip.sublime-syntax

```diff
@@ -56,10 +55,10 @@ contexts:
       captures:
         1: punctuation.definition.backslash.tex
       push: file-argument
-    - match: ((\\)(:?generate|needed)){{endcs}}
+    - match: (\\)(:?generate|needed){{endcs}}
```

Comment:
Typo?

```suggestion
    - match: (\\)(?:generate|needed){{endcs}}
```

Reply:
seems like, and then copied all over the file :(